### PR TITLE
Editorial: Document parameter type for Date Abstract Operations

### PR DESCRIPTION
**Pull request by gibson042**
A bigger refactoring of the Date AOs is still in order, but this will help comprehensibility in the meantime.

### CODE REVIEW

**Review by bakkot — APPROVED** [review]

**Review by WeslleyFratini — APPROVED** [review]

**Comment by michaelficarra** [comment]
While we're at it, the `tv` parameter of `TimeString`, `DateString`, `TimeZoneString`, and `ToDateString` is a `Number`.

File: spec.html [hunk 1]
```diff
@@ -30604,12 +30604,12 @@ <h1>
...
         <h1>
           UTC (
-            _t_: unknown,
+            _t_: a Number,
           )
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It converts _t_ from local time to UTC.</dd>
+          <dd>It converts _t_ from local time to a UTC time value.</dd>
```
Comment (michaelficarra) [inline spec.html:30612]: I would prefer this wording to be more similar to the wording above:

> It converts time value _t_ from UTC to local time.

Comment (michaelficarra) [reply spec.html:30612]: What does "from local time" add here?

Comment (gibson042) [reply spec.html:30612]: It communicates the semantics of the value, albeit poorly (as noted in the PR description, a bigger refactoring of the Date AOs is still in order). But specifically, _t_ is a Number representing a local date and time of day equal to the UTC date and time of day that correspond with (mis)interpreting it as a time value and ignoring the bounds that normally apply.

File: spec.html [hunk 2]
```diff
@@ -30585,12 +30585,12 @@ <h1>
...
         <h1>
           LocalTime (
-            _t_: unknown,
+            _t_: a Number,
           )
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It converts _t_ from UTC to local time.</dd>
+          <dd>It converts time value _t_ from UTC to local time.</dd>
```
[the remark this answers is not in the document]
Comment (bakkot) [reply spec.html]: I went ahead and pushed this change.

**Review by michaelficarra — APPROVED** [review]

**Pull request merged**